frontend/packages/ux-editor/src/components/config/expressions/DataSourceValue.tsx

```diff
@@ -33,6 +32,7 @@ export const DataSourceValue = ({
   const dataModelElementsData = datamodelQuery?.data ?? [];
   const formLayoutsData = formLayoutsQuery?.data ?? [];
 
+  // TODO: Make sure all data model fields are included - what if there are multiple data models?
```

Comment:
Is this TODO something you want to discuss within this PR? If not, should we create a separate issue for this todo? :😊

Reply:
This todo was something I wanted feedback on yes. I struggle to see if all feilds are retrieved. But if it is okay to leave it like this for the first beta release I guess we can fix it in a new issue